src/main/java/net/syscon/elite/web/config/AuthAwareAuthenticationToken.java

```diff
@@ -9,16 +10,12 @@
 
 @Getter
 public class AuthAwareAuthenticationToken extends JwtAuthenticationToken {
-    private final String authSource;
+    private final AuthSource authSource;
     private final Object principal;
 
     public AuthAwareAuthenticationToken(final Jwt jwt, final Object principal, final String authSource, final Collection<? extends GrantedAuthority> authorities) {
         super(jwt, authorities);
-        this.authSource = authSource != null ? authSource : "nomis";
```

Comment:
We no longer default to a Nomis user as this is misleading.

Reply:
:+1: